templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -201,23 +201,42 @@
           "archiveHierarchyFolders.$": "$.archiveHierarchyFolders",
           "contentFolders.$": "$.contentFolders"
         },
-        "contentAssets.$": "$.contentAssets"
+        "contentAssets.$": "$.contentAssets",
+        "assetBucket.$": "$.asset.bucket",
+        "assetKey.$": "$.asset.key",
+        "foldersBucket.$": "$.folders.bucket",
+        "foldersKey.$": "$.folders.key"
       }
     },
     "Merge array": {
       "Type": "Pass",
       "Next": "Map over each Folder Id",
       "Parameters": {
         "allFolders.$": "$.allFolders.[*]*",
+        "assetsBucket.$": "$.assetsBucket",
+        "assetsKey.$": "$.assetsKey",
         "contentAssets.$": "$.contentAssets",
         "executionId.$": "$$.Execution.Name",
+        "foldersBucket.$": "$.foldersBucket",
+        "foldersKey.$": "$.foldersKey",
         "workflowContextName": "Ingest OPEX (Incremental)",
         "stagingPrefix.$": "States.Format('opex/{}', $$.Execution.Name)",
         "stagingBucket": "${ingest_staging_cache_bucket_name}"
       }
     },
```

Comment:
We should be able to remove the `Convert array to object` and `Merge array` states as these were used to join the `archiveFolders` and `contentFolders` arrays, which we've now already done within the Mapper Lambda when generating `folders.json`.

Reply:
Oh yeah, Cool